remove checking for imlib, pixbuf, gtkgl and libglade

#
# Patch managed by http://www.holgerschurig.de/patcher.html
#

--- pygtk-0.6.12/configure.in~remove-imlib-et-al
+++ pygtk-0.6.12/configure.in
@@ -45,17 +45,14 @@
 fi
 
 AM_PATH_GTK(gtk_required_version,,,$extra_mods)
-AM_PATH_GDK_IMLIB(imlib_required_version, build_imlib=true, build_imlib=false)
-AM_CONDITIONAL(BUILD_IMLIB, $build_imlib)
 
-AM_PATH_GDK_PIXBUF(pixbuf_required_version, build_pixbuf=true, build_pixbuf=false)
-AM_CONDITIONAL(BUILD_PIXBUF, $build_pixbuf)
+AM_CONDITIONAL(BUILD_IMLIB, false)
 
-AM_PATH_GTKGL(build_gtkgl=true, build_gtkgl=false)
-AM_CONDITIONAL(BUILD_GTKGL, $build_gtkgl)
+AM_CONDITIONAL(BUILD_PIXBUF, false)
 
-AM_PATH_LIBGLADE(build_libglade=true, build_libglade=false)
-AM_CONDITIONAL(BUILD_LIBGLADE, $build_libglade)
+AM_CONDITIONAL(BUILD_GTKGL, false)
+
+AM_CONDITIONAL(BUILD_LIBGLADE, false)
 
 
 AC_ARG_ENABLE(numpy,
